src/components/Handlers/withHandlers.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+function getDisplayName(WrappedComponent) {
+  return WrappedComponent.displayName || WrappedComponent.name || 'Component';
+}
+
+function getComponentFromHandler(event, stripes, module, data) {
+  const m = module.getModule();
+  const eventHander = m[module.handlerName];
```

Comment:
(I should have asked this when reviewing the previous PR, but don't forget to document `handlerName` in the Stripes developer guide.)